src/Renderer/Model/Model.h

```diff
@@ -0,0 +1,92 @@
+#pragma once
```

Comment:
avoid using `#pragma once` since there's still no guarantee it works across compilers